CMakeLists.txt

```diff
@@ -108,6 +108,10 @@ if("${ARCH}" MATCHES "arm" OR "${ARCH}" MATCHES "aarch64")
     option(WITH_NEON "Build with NEON intrinsics" ON)
 endif()
 
+if("${ARCH}" MATCHES "s390x")
+    option(WITH_DFLTCC "Build with DEFLATE CONVERSION CALL" OFF)
```

Comment:
This should mention that it is IBM Z/s390x, like it does in the configure help text